packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -273,6 +273,10 @@ export const actionCreators: ActionCreators = {
         } else {
           updatedWorkspace = await devWorkspaceClient.changeWorkspaceStatus(workspace, true, true);
         }
+        await devWorkspaceClient.onStart(
+          updatedWorkspace,
+          getState().dwPlugins.defaultEditorName as string,
```

Comment:
I'm not sure if the default editor will always fit here. If I got it correctly, you need somehow to figure out which che-editor was used on the workspace creation. Or, what may be easier, store che-editor as devworkspace annotation when it's created, so then you'll be able to get it on the devworkspace start.

Reply:
> If I got it correctly, you need somehow to figure out which che-editor was used on the workspace creation

Yes, that was the idea, and sounds good, my latest push sets an annotation for the che-editor ID:
https://github.com/eclipse-che/che-dashboard/blob/ebf10c803c3a80d1ecf511a146718bcdee0981c4/packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts#L466-L471

![image](https://user-images.githubusercontent.com/83611742/146657918-97682aa0-5c97-4ada-ac2e-065be95dd3ce.png)